docs/iris/src/userguide/code_maintenance.rst

```diff
@@ -0,0 +1,38 @@
+Code Maintenance
```

Comment:
I think this is covered in the white paper.  This section could be removed, to try and keep the user guide more tightly focussed